csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -151,22 +141,14 @@ public static SessionOptions MakeSessionOptionWithNupharProvider(String settings
         /// A helper method to construct a SessionOptions object for ROCM execution.
         /// Use only if ROCM is installed and you have the onnxruntime package specific to this Execution Provider.
         /// </summary>
-        /// <returns>A SessionsOptions() object configured for execution on deviceId=0</returns>
-        public static SessionOptions MakeSessionOptionWithRocmProvider()
-        {
```

Comment:
Also pointless.

Reply:
Wouldn't it require someone to update their source code if they were using this method in the next release ?

Reply:
I wouldn't have thought so. 

If they recompile it will resolve to the method with a default arg (which is why the separate method with no args is pointless). 
If they don't recompile, they'll still be calling the entry point with a deviceId of 0.